client/modules/death.lua

```diff
@@ -42,10 +42,11 @@ local firstSpawn = true
 if GetResourceState("spawnmanager"):find("start") then
     exports.spawnmanager:setAutoSpawnCallback(function()
         if not firstSpawn then
```

Comment:
I had rushed the bridge tbh and didn’t test. This is supposed to just be `If not firstSpawn then return end` in one line.